bugsnag-android-core/src/main/java/com/bugsnag/android/Stackframe.kt

```diff
@@ -5,7 +5,7 @@ import java.io.IOException
 /**
  * Represents a single stackframe from a [Throwable]
  */
-class Stackframe @JvmOverloads internal constructor(
+class Stackframe @JvmOverloads constructor(
```

Comment:
this should remain internal - we never want to expose constructors on classes within the `Event` payload to users, as it's an implementation detail and will constrain future changes.

Reply:
I had to open it up so that the ANR tests could see it. Is there another way to make it visible?

Reply:
I've made this constructor internal now. The other constructor is public, but only accepts a NativeStackframe, so it's pretty safe I think.